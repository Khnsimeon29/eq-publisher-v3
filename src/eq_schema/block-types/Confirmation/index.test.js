const Confirmation = require(".");

describe("Confirmation", () => {
  it("should build valid runner Confirmation", () => {
    const confirmation = new Confirmation();
    expect(confirmation).toMatchSnapshot();
  });
});
