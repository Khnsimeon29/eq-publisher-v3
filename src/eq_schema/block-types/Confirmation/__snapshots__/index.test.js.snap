// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Confirmation should build valid runner Confirmation 1`] = `
Confirmation {
  "blocks": Array [
    Object {
      "content": Object {
        "contents": Array [
          Object {
            "list": Array [
              "You will not be able to access or change your answers on submitting the questionnaire",
              "If you wish to review your answers please select the relevant completed sections",
            ],
            "title": "You are now ready to submit this survey",
          },
        ],
        "title": "Submission",
      },
      "id": "confirmation",
      "type": "Confirmation",
    },
  ],
  "id": "confirmation-group",
  "title": "confirmation",
}
`;
